Submodule Telegram/lib_base contains modified content
diff --git a/Telegram/lib_base/base/platform/linux/base_linux_glibmm_helper.h b/Telegram/lib_base/base/platform/linux/base_linux_glibmm_helper.h
index 864fe53..613905e 100644
--- a/Telegram/lib_base/base/platform/linux/base_linux_glibmm_helper.h
+++ b/Telegram/lib_base/base/platform/linux/base_linux_glibmm_helper.h
@@ -10,6 +10,7 @@
 
 #include <glibmm/variant.h>
 
+#if 0
 namespace Glib {
 
 struct Int64Helper0 {};
@@ -107,6 +108,7 @@ public:
 };
 
 } // namespace Glib
+#endif
 
 namespace base {
 namespace Platform {
